oemoflex/facades.py

```diff
@@ -454,7 +454,7 @@ def __init__(self, *args, **kwargs):
 
         self.expandable = bool(kwargs.get("expandable", False))
 
-        self.initial_storage_level = kwargs.get("initial_filling_level")
+        self.initial_storage_level = kwargs.get("initial_storage_level")
```

Comment:
Does this parameter have to be passed explicitly, or is it already an attribute of the parent class?

Reply:
Done in 4152cac